Library/Homebrew/cmd/deps.rb

```diff
@@ -109,11 +113,21 @@ def deps
       end
 
       if args.graph?
-        dot_code = dot_code(dependents, recursive: recursive, args: args)
+        dep_graph = {}
+        dependents.each do |d|
+          graph_deps(d, dep_graph: dep_graph, recursive: recursive, args: args)
+        end
```

Comment:
This works okay but I think I would prefer something like this where the result of the method is stored in `dep_graph`:

```suggestion
        dep_graph = graph_deps(dependents, recursive: recursive, args: args)
```

I don't think this makes `graph_deps` a lot more complicated since really it just needs the `dependents.each` to be moved inside